### ISSUE TITLE
Add Fields to Fetch and Add Content to BookItemBack

### ISSUE THREAD

**Issue by livoszlak** [livoszlak/book-finder-api-2.0#8]
In the bare-bones fetch of [useFetchBooks](https://github.com/livoszlak/book-finder-api-2.0/blob/main/src/hooks/useFetchBooks.js) there are many fields that are not used in the frontend. For example, categories, pageCount, description, et c. The [BookItemBack](https://github.com/livoszlak/book-finder-api-2.0/blob/main/src/components/BookItemBack.jsx) is obviously a small surface to display all of that on - but maybe you have a good solution for it?

Suggestions:

- [ ] 1. Move title and author/-s to [BookItem](https://github.com/livoszlak/book-finder-api-2.0/blob/main/src/components/BookItem.jsx) - but please make sure both will fit above or beneath cover image, in a way that doesn't mess with the styling cohesion. Title and author/-s should have the same space on every BookItem card, and cover image should also maintain consistent space on the card. In v1 of this project, I solved the issue of different lengths of titles and author names with having them "disappear" under ellipses, but perhaps you have a better solution?
- [ ] 2. Having moved title and author from the back view of the card, display Page count and categories there instead. And perhaps description can appear in an additional BookItemBack component if user clicks a "See description"-button? Check out [Nested modal](https://mui.com/material-ui/react-modal/) - is it possible to have the child modal appear next to the parent modal? Side by side, sliding in from right? Let's find out!
- [ ] 3. If "See description" child modal is implemented, perhaps replace the link icon on the back view of the parent modal with a "View on Google Books" button?

**Comment by om9090** [comment]
@livoszlak Please assign the issue to me.

I understood all the changes proposed by you. I would love to contribute in your project. I have experience of over 1.5 years in Frontend Development I can defiinitely solve your issue. I will make the website more interactive and will enhace the UI and UX.

Thank you.

**Comment by livoszlak** [comment]
@om9090 Absolutely! Thank you for wanting to contribute. Assigning this issue to you right now! 🌷

**Comment by om9090** [comment]
Thank you so much. I’ll start working on it.
